scripts/con_spec.js

```diff
@@ -28,27 +28,22 @@ function (context, args) { // T2 CON_SPEC passes in {s:"string", d:"digit"}
 //
 // Ex: npc.loc{CON_SPEC:#s.user.con_spec}
 //
-
+    
     // Ensure that args were passed, and that those args are exactly
     // d:<digit> and s:<string>. Anything else, show the usage screen.
-    if (!args
-        || !((args.d && args.d.constructor == Number)
-        &&   (args.s && args.s.constructor == String))) {
+    try {
+        if(!args.d)
+            throw "D != #"
+        return (args.s.match(new RegExp(args.d,"g")) || []).length
+    } catch(e) {
         return {ok:false, msg:`
 Tier 2 \`DCON_SPEC\` lock scriptor
-
```

Comment:
These newlines actually affect the output in Hackmud, they didn't need to be removed.